doc/source/index.rst

```diff
@@ -18,9 +19,13 @@ questions, feel free to reach out at the `issues page`_.
 Creating your JupyterHub
 ------------------------
 
+This is a short walkthrough starting from square one. Go from having
```

Comment:
This tutorial starts from "step zero" and walks through how to install and configure a complete JupyterHub deployment in the cloud. Using Kubernetes and the JupyterHub Helm chart will provide sensible defaults for an initial deployment.

To get started, go to :ref:`create a Kubernetes cluster on Google Cloud <google-cloud>`.